backend/server.ts

```diff
@@ -1,19 +1,23 @@
 import { BACKEND_PORT } from "./config";
-import { schema } from "./graphql/schema";
+import { dbUrl } from "./psql";
+const { postgraphile } = require("postgraphile");
 const express = require("express");
 const cors = require("cors");
 const app = express();
-const { graphqlHTTP } = require("express-graphql");
+
 app.use(cors());
 
 app.use(
-	"/api",
-	graphqlHTTP({
-		schema: schema,
+	postgraphile(dbUrl, "public", {
+		ignoreRBAC: true,
+		extendedErrors: ["errcode", "detail", "hint"],
 		graphiql: true,
+		enhanceGraphiql: true,
+		graphqlRoute: "/api",
```

Comment:
Should we as `/api` and `/graphiql` as constant? I not sure my suggestion is good. How you think about it?

Reply:
this is used only in one place it's a standard endpoint. I would leave as is.